iocage/lib/Jail.py

```diff
@@ -417,27 +418,41 @@ def _revert_jail_started():
 
         jailExecEvent.add_rollback_step(_revert_jail_started)
 
-        child, stdout, stderr = self.exec(command, ignore_error=True)
-        if child.returncode != 0:
-            if (error_handler is None):
-                passed = False
-                message = ""
+        if passthru is True:
+            try:
+                self.passthru(command)
```

Comment:
Passthru enables CLI users to interact with an executed command.

Reply:
given how long and unreadable this if/else is, maybe we should also bundle the else's call into a separate function like `passthru`